MSAL/test/integration/native_auth/end_to_end/MSALNativeAuthEndToEndBaseTestCase.swift

```
@@ -99,27 +100,22 @@ class MSALNativeAuthEndToEndBaseTestCase: XCTestCase {
                 format: customAuthorityURLFormat
             )
             
-            let authority = try? MSALCIAMAuthority(
+            let authority = try! MSALCIAMAuthority(
                 url: URL(string: customSubdomain)!,
                 validateFormat: false
             )
             
-            let configuration = MSALPublicClientApplicationConfig(
-                clientId: clientId,
-                redirectUri: nil,
-                authority: authority
-            )
-
+            let configuration = MSALNativeAuthPublicClientApplicationConfig(clientId: clientId, authority: authority, challengeTypes: challengeTypes)
+            configuration.capabilities = capabilities
+           
             return try? MSALNativeAuthPublicClientApplication(
-                configuration: configuration,
-                challengeTypes: challengeTypes
+                nativeAuthConfiguration: configuration
```

Using `try?` silently swallows initialization errors

How about
```
do {
    return try MSALNativeAuthPublicClientApplication(nativeAuthConfiguration: configuration)
} catch {
    XCTFail("Failed to create MSALNativeAuthPublicClientApplication: \(error)")
    return nil
}
```

---

It's another approach that makes the test longer (do/catch is at least 4 lines). In case the initialisation fails, the test will fail. Then we can debug. I prefer the current approach for testing